DeveloperConsole.cs

```diff
@@ -40,13 +40,23 @@ private void CommandRunner (string command)
                     result = "Commands:\n" +
                         "help - Displays this message\n" +
                         "clear - Clears the command history\n" +
-                        "set - Sets a value\n" +
-                        "set health <value> - Sets the player's health\n";
+                        "set - Shows the commands available for set\n";
                     break;
+
+                // Set command
                 case "set":
```

Comment:
This section is for the command control, not the help section. If the intention is to add a help section for set commands, please create a secondary switch case inside help